doc/WebSite/Entity-History.md

```diff
@@ -96,18 +96,18 @@ save change logs for a desired property.
 entity**. Thus, you can **hide sensitive data** in change logs, such as
 passwords for example.
 
-### Reason
+### Reason Property
 
-Entity change set has a **Reason** property that can be used to understand why a
-set of changes had occurred, i.e. the use case that resulted in these changes.
+The entity change set has a **Reason** property that can be used to understand why a
+set of changes have occurred, i.e. the use case that resulted in these changes.
```

Comment:
"a set (of changes) `has` occurred" since "a set" is singular.
- https://www.google.com.sg/search?q="a+set+of+changes+has" (65,800 results)
- https://www.google.com.sg/search?q="a+set+of+changes+have" (4 results)

Reply:
How about "set of changes occurred"

Reply:
"`has` occurred" reads better.

I blame error messages: https://www.google.com.sg/search?q=has+occurred

Reply:
"changes" is plural, though.

Reply:
Singular: has
Plural: have

Reply:
"changes" is plural, no doubt about that. It should be a little more obvious with word repetition:
- "a deck of cards has 52 cards" not "a deck of cards have 52 cards"
- "a deck has 52 cards"

Reply:
Gotcha.  Just checked with my mother (English teacher).  She's saying grammatically, you are correct. :sweat_smile:

Reply:
Because it is "one set"